share/package.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env sh
+
+if [ $# -eq 0 ]
+  then
+    echo "Path to package need to be specify"
+	exit 1
+fi
+
+if [[ `mkdir -p "work/gradle/wrapper"` -ne 0 ]];
+	then
+        echo "Cannot create work folder"
+        exit 1
+fi
+
+baseUrl=https://raw.githubusercontent.com/Cognifide/gradle-aem-plugin/master/share/wrapper
+declare -a arr=("gradlew" "build.gradle" "settings.gradle" "gradle/wrapper/gradle-wrapper.properties" "gradle/wrapper/gradle-wrapper.jar")
```

Comment:
I know alternative syntax of declaring arrays... please ensure that one which is more standard

```
 myArray=(
        "a"
        "b"
         "c"
        )
```